### PR TITLE
Added kit documentations - Group 4

### PR DESCRIPTION
**Pull request by christinaatai**
#### Breaking Changes

_Please reflect if your changes may break in some way (changes like removing/renaming props are examples of breaking changes)_

#### Checklist:

- [ ] **SPECS** Please cover your changes with specs
- [x] **CHANGELOG** Please add an entry on `[Unreleased]` section to every functionality `ADDED`/`CHANGED`/`DEPRECATED`/`REMOVED`/`FIXED`

### CODE REVIEW

File: app/pb_kits/playbook/pb_progress_simple/docs/_footer.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+### Things to Avoid
+Do not use as turbo bar.
```
Comment (thestephenmarshall) [inline app/pb_kits/playbook/pb_progress_simple/docs/_footer.md]: Does this mean "Turbolinks bar"?

Comment (christinaatai) [reply app/pb_kits/playbook/pb_progress_simple/docs/_footer.md]: Yes! I will fix that

**Comment by christinaatai** [comment]
**Progress simple**
Screen Shot 2020-03-03 at 3 00 53 PM
Screen Shot 2020-03-03 at 3 00 59 PM

**Comment by christinaatai** [comment]
**Progress Simple**

Screen Shot 2020-03-03 at 3 02 07 PM
Screen Shot 2020-03-03 at 3 03 09 PM

**Comment by christinaatai** [comment]
**Source**

Screen Shot 2020-03-03 at 3 55 45 PM

**Comment by christinaatai** [comment]
**Star Rating**

Screen Shot 2020-03-03 at 4 31 16 PM

**Comment by christinaatai** [comment]
**Stat Change**

Screen Shot 2020-03-03 at 4 31 58 PM

**Comment by christinaatai** [comment]
**Table**

Screen Shot 2020-03-03 at 4 43 02 PM
Screen Shot 2020-03-03 at 4 43 09 PM
Screen Shot 2020-03-03 at 4 43 17 PM

**Comment by christinaatai** [comment]
**Body**

Screen Shot 2020-03-04 at 11 44 33 AM
Screen Shot 2020-03-04 at 11 44 40 AM

**Comment by christinaatai** [comment]
**Caption**

Screen Shot 2020-03-04 at 11 56 33 AM
Screen Shot 2020-03-04 at 11 56 39 AM

**Comment by christinaatai** [comment]
**Title**

Screen Shot 2020-03-04 at 12 01 04 PM

**Comment by christinaatai** [comment]
**Contact**

Screen Shot 2020-03-04 at 12 02 02 PM

**Comment by christinaatai** [comment]
**Currency**

Screen Shot 2020-03-04 at 12 11 11 PM

**Comment by christinaatai** [comment]
**Dashboard Value**

Screen Shot 2020-03-04 at 12 15 52 PM

**Comment by christinaatai** [comment]
**Date Range Inline**

Screen Shot 2020-03-04 at 12 16 34 PM

**Comment by christinaatai** [comment]
**Date Stacked**

Screen Shot 2020-03-04 at 12 17 20 PM
Screen Shot 2020-03-04 at 12 17 32 PM

File: app/pb_kits/playbook/pb_source/docs/_description.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Source describes how a customer discovered PHRG.
```
Comment (jasoncypret) [inline app/pb_kits/playbook/pb_source/docs/_description.md]: Remove PHRG and rephrase to cover photo please industries.

Comment (christinaatai) [reply app/pb_kits/playbook/pb_source/docs/_description.md]: @jasoncypret Updated source kit screenshot

Comment (jasoncypret) [reply app/pb_kits/playbook/pb_source/docs/_description.md]: I don't see that this change was push up

**Comment by jasoncypret** [comment]
@christinaatai your PR has changes you didn't make included in it. I believe you must have had some rebasing problems. A quick fix might be to run `git checkout path/to/file` on each file to reset it. Then you can run `git -amend` and then force push back up. Another option could be to cherry pick...but you might reach out to @thestephenmarshall on that

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,10 +6,6 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 ### Unreleased
```
Comment (jasoncypret) [inline CHANGELOG.md]: I think you removed Megans + yours

**Pull request merged**